### ISSUE TITLE
2 high severity vulnerabilities

### ISSUE THREAD

**Issue by trevorphillipscoding** [discordjs/discord.js#7319]
### Which package is this bug report for?

discord.js

### Issue description

1. run `npm audit`

This pops up:
```bash
# npm audit report

node-fetch  <3.1.1
Severity: high
node-fetch is vulnerable to Exposure of Sensitive Information to an Unauthorized Actor - https://github.com/advisories/GHSA-r683-j2x4-v87g
fix available via `npm audit fix --force`
Will install discord.js@11.6.4, which is a breaking change
node_modules/node-fetch
  discord.js  >=12.0.0
  Depends on vulnerable versions of node-fetch
  node_modules/discord.js

2 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

### Package version

13.6.0

### Node.js version

16.13.2

### Operating system

OSX/Mac M1 Chip

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

No Partials

### Which gateway intents are you subscribing to?

GUILDS, GUILD_MEMBERS, GUILD_BANS, GUILD_EMOJIS_AND_STICKERS, GUILD_INTEGRATIONS, GUILD_WEBHOOKS, GUILD_INVITES, GUILD_VOICE_STATES, GUILD_PRESENCES, GUILD_MESSAGES, GUILD_MESSAGE_REACTIONS, GUILD_MESSAGE_TYPING, DIRECT_MESSAGES, DIRECT_MESSAGE_REACTIONS, DIRECT_MESSAGE_TYPING, GUILD_SCHEDULED_EVENTS

### I have tested this issue on a development release

c10afea

**Comment by KhafraDev** [comment]
Non-issue & not currently fixable unless.

node-fetch maintainers said they would backport bug fixes to v2.x so you gotta wait for that to happen I guess.

**Comment by mtripg6666tdr** [comment]
it seems this security issue has been already fixed in v2.6.7
https://github.com/node-fetch/node-fetch/releases/tag/v2.6.7

**Comment by YodaLightsabr** [comment]
Shouldn't this be looked at again as there's a new version of node-fetch?
